dist/templates/rbac-ovnkube-master.yaml.j2

```diff
@@ -101,6 +101,7 @@ rules:
           - egressqoses
           - egressservices/status
           - adminpolicybasedexternalroutes/status
+          - egressqoses/status
```

Comment:
is this for non-ic? how does arch look there, is it that CM does the status patch but Master does the rest of the logic for EQoS?

Reply:
yes, this is for non-ic case, this is also needed for ovnkube-master container to update status message at a 'global' zone level.